MANIFEST.in

```diff
@@ -1,3 +1,4 @@
 recursive-include cpp *.cc *.h
 recursive-include extern *
+recursive-include freud *.py *.pyx *.pxd *.pxi
```

Comment:
Shouldn't the `*.py` files be included by default. Is it included just to be safe?

Reply:
Yes, it's included by default - just wanted that to be explicitly clear. https://packaging.python.org/guides/using-manifest-in/#how-files-are-included-in-an-sdist